docs/Community/Downloads.md

```diff
@@ -32,11 +32,13 @@ It is essential that you verify the integrity of release downloads. See [instruc
 From [Maven Central](https://search.maven.org/search?q=g:%20org.apache.datasketches).
 
 ## Version Numbers
-Apache DataSketches uses [semantic versioning](https://semver.org/). Version numbers use the form major.minor.incremental and are incremented as follows:
+Apache DataSketches uses [semantic versioning](https://semver.org/). Version numbers use the form major.minor.incremental and are updated as follows:
 
-* __major version__ for major new functionality and/or major API changes that may be incompatible with prior versions
-* __minor version__ for new functionality and scheduled bug fixes. This should be API compatible with prior versions
-* __incremental version__ for unscheduled bug fixes only
+* __major version__ for major new functionality and/or major API changes that may (or may not) be API compatible with prior versions. 
```

Comment:
For a version bump, users should expect there to be some API-incompatible changes. I think this language makes it too soft.